lib/Doctrine/Sequence/Mysql.php

```diff
@@ -83,7 +83,7 @@ public function nextId($seqName, $onDemand = true)
      * @param string  $table name of the table into which a new row was inserted
      * @param string  $field name of the field into which a new row was inserted
      *
-     * @return int|string
+     * @return int|string|false
      */
     public function lastInsertId($table = null, $field = null)
```

Comment:
Apparently `lastInsertId` may return `false`, I looked in php's source, and this is true.

PHPStan complained about this, but only in PHP 8.